### PR TITLE
528 bootstrap navs

### PR DESCRIPTION
**Pull request by Delnote**

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/AlignmentTests.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+    @Test(dataProvider = "clickValidate")
+    public void linkClickableATests(int index, String pageTitle) {
+        navsAlignmentEnd.navItemLink.get(index).highlight();
+        navsAlignmentEnd.navItemLink.get(index).click();
+        newWindowTitleCheck(pageTitle);
+        navsAlignmentEnd.navItemLink.get(index).unhighlight();
+    }
+
+    private void newWindowTitleCheck(String pageTitle) {
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/AlignmentTests.java]: put it into some helper class (it's too much repeats)

File: jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/PillsTests.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        navsPills.navItemLink.get(index)
+                .is()
+                .core()
+                .hasClass("nav-link")
+                .text(is(linkText));
+    }
+
+    @Test(dataProvider = "clickValidate")
+    public void linkClickableLiTests(int index, String pageTitle) {
+        navsPills.navItem.get(index).highlight();
```
Comment (mf-code) [inline jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/PillsTests.java]: perhaps, to use chaining here is better:
```
navsPills.navItem.get(index)
        .highlight();
        .click();
```

Comment (mf-code) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/PillsTests.java]: or we should to create a local variable like: MyType myVar=navsPills.navItem.get(index);

Comment (Delnote) [reply jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/navs/PillsTests.java]: .highlight().click() - так не работает. Только одно действие на элемент.

**Pull request merged**